src/types/Constants.ts

```diff
@@ -203,6 +203,7 @@ export enum ROLES {
   IDP_DISABLE = 'disable_idp',
   MODIFY_USER_ACCOUNT = 'modify_user_account',
   ORGANIZATION_VIEW = 'view_organization',
+  MY_ORGANIZATION_VIEW = 'view_company_data',
```

Comment:
Quick question - for what do we use the value in line 205?
Can we get rid of it?
@nidhigarg-bmw  any opinion?
From R&R perspective this permission is not even existing 

@oyo  FYI

Reply:
the line was unused so I updated the PR removing it